resources/source-types/oracledb.yaml

```
@@ -115,9 +224,41 @@ spec:
               - '{{ $fp }}'
               {{ end }}
             {{ end }}
+      {{ end }}
+
+    processors: |
+      {{ if .enable_logs }}
+      - resourcedetection/logs:
+          detectors: ["system"]
+          system:
+            hostname_sources: ["os"]
+      {{ end }}
+
+  metrics:
+    receivers: |
+      {{ if .enable_metrics }}
+      - plugin/oracledb_metrics:
+          path: $OIQ_OTEL_COLLECTOR_HOME/plugins/oracledb_metrics.yaml
+          parameters:
+            endpoint: {{ .endpoint }}
+            username: {{ .username }}
+            password: {{ .password }}
```

We might want to single quote this to avoid weird issues with characters 
```suggestion
            password: '{{ .password }}'
```